package.json

```diff
@@ -32,9 +32,8 @@
     "precheck": "twolfson-style precheck lib/ test/",
     "lint": "twolfson-style lint lib/ test/",
     "pretest": "npm run build",
-    "test": "npm run verify-webdriver && mocha --timeout 10000 --reporter dot && npm run lint",
-    "verify-webdriver": "curl --silent http://localhost:4444/ > /dev/null || (echo \"Selenium server wasn't started. Please start it via \\`npm run webdriver-manager-start\\`.\" 1>&2 && exit 1)",
-    "webdriver-manager-start": "webdriver-manager start"
+    "test": "mocha --timeout 30000 --reporter dot && npm run lint",
```

Comment:
Please add back the `npm run verify-webdriver`. If we want to support Windows, then we should add a `npm run test-windows`, not depreciate the developer friendly features